docs/modules/ROOT/pages/reference/extensions/joor.adoc

```diff
@@ -74,6 +74,12 @@ The specific location of the configuration of the jOOR language.
 The specific default result type of an expression expressed in jOOR language.
 | `string`
 | 
+
+|icon:lock[title=Fixed at build time] [[quarkus.camel.joor.compile-ahead-of-time]]`link:#quarkus.camel.joor.compile-ahead-of-time[quarkus.camel.joor.compile-ahead-of-time]`
```

Comment:
Wonder if `quarkus.camel.joor.compile-at-build-time` would be better? I think it's more consistent with the nomenclature that Quarkus uses (I.e AOT is rarely mentioned in docs. It's usually 'build time') .

I'm nitpicking a bit here, so happy to be ignored if you disagree.

Reply:
👍 It makes sense indeed, let me rename it